highlight.js

```diff
@@ -0,0 +1,50 @@
+const refractor = require('refractor/core.js');
```

Comment:
This plugin should be moved to another place `src/utilities/remark/plugins` would be a good place, we could also create a package from it since it doesn't exist.